subsys/bluetooth/services/fast_pair/include/fp_crypto.h

```diff
@@ -103,6 +105,87 @@ size_t fp_crypto_account_key_filter_size(size_t n);
 int fp_crypto_account_key_filter(uint8_t *out, const struct fp_account_key *account_key_list,
 				 size_t n, uint8_t salt);
 
+/** Encrypt data using AES-CTR.
+ *
+ * @param[out] out Buffer to receive encrypted data.
+ * @param[in] in Plaintext data.
+ * @param[in] data_len Length of data (in bytes).
+ * @param[in] k 128-bit (16-byte) AES key.
+ * @param[in] nonce 64-bit (8-byte) nonce.
+ *
+ * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int fp_crypto_aes_ctr_encrypt(uint8_t *out, const uint8_t *in, size_t data_len, const uint8_t *k,
```

Comment:
Maybe k->key, the parameter name will be still short and its name told us what it is exactly. Consider this also in other API
```suggestion
int fp_crypto_aes_ctr_encrypt(uint8_t *out, const uint8_t *in, size_t data_len, const uint8_t *key,
```

Reply:
You can consider also to make other parameters name more descriptive